values.yaml

```diff
@@ -90,6 +98,25 @@ global:
   registry:
      bucket: ${PROJECT_ID}-gitlab-registry
 
+  ## https://docs.gitlab.com/charts/installation/deployment#outgoing-email
+  ## Outgoing email server settings
+  smtp:
+    enabled: "${$ENABLE_SMTP}"
+    user_name: "${SMTP_USER}"
+    password: 
+      secret: {$SMTP_SECRET_NAME}
```

Comment:
Are you sure about this syntax (`{$<varname>}`)?

Reply:
I'm a F**** idiot.